mne/epochs.py

```diff
@@ -402,6 +409,8 @@ def decimate(self, decim, copy=False):
         else:
             epochs._decim_slice = decim_slice
             epochs.times = epochs._raw_times[epochs._decim_slice]
+        epochs.tmin = epochs.times[0]
+        epochs.tmax = epochs.times[-1]
```

Comment:
this has consequences and is a change of behavior. how was it before? why is it necessary? tmin and tmax basically tell you how it was clipped in the time axis in the first place.